packages/delphi/notes.txt

```diff
@@ -0,0 +1,21 @@
+$(BDSCOMMONDIR)\Bpl
```

Comment:
I'm afraid these notes are not really decipherable, even when I know some comments about what you're doing. Please remove them from this PR. And / or reformat the useful bits and place in packages/delphi/README.md in Markdown format.